.rubocop_todo.yml

```diff
@@ -1,11 +1,17 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2021-01-29 12:53:33 UTC using RuboCop version 1.9.0.
+# on 2021-02-11 05:02:06 UTC using RuboCop version 1.9.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+# Cop supports --auto-correct.
+Lint/RedundantDirGlobSort:
```

Comment:
Keeping this one by now as on Ruby 3.0 this cop https://docs.rubocop.org/rubocop/cops_lint.html#lintredundantdirglobsort raises
and in 2.x this other cops https://docs.rubocop.org/rubocop/cops_lint.html#lintnondeterministicrequireorder raises

Reply:
Brilliant 😂